### PR TITLE
entitlement management: introduce SOD relationships to v1.0

### PR DESCRIPTION
**Pull request by markwahl-msft**
Separation of duties with incompatible access packages and groups is an existing feature of entitlement management that previously has only been surfaced in the beta API.

### CODE REVIEW

File: api-reference/v1.0/toc.yml [hunk 1]
```diff
@@ -5618,6 +5618,8 @@ items:
               href: api/entitlementmanagement-list-assignments.md
             - name: Get
               href: api/accesspackageassignment-get.md
+            - name: Additional access
```
Comment (FaithOmbongi) [inline api-reference/v1.0/toc.yml:5621]: Precede with a verb
```suggestion
            - name: Check additional access
```

File: api-reference/v1.0/api/accesspackageassignment-additionalaccess.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+When a result set spans multiple pages, Microsoft Graph returns that page with an `@odata.nextLink` property in the response that contains a URL to the next page of results. If that property is present, continue making additional requests with the `@odata.nextLink` URL in each response, until all the results are returned. For more information, see [paging Microsoft Graph data in your app](/graph/paging).
+
+## Examples
+
+The following example gets the access package assignments for users who have assignments to both access packages.
+
+### Request
+
+
+# [HTTP](#tab/http)
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/accesspackageassignment-additionalaccess.md:67]: Will be inserted by the snippets generator. Currently distorts the page.
```suggestion
```

File: changelog/Microsoft.IGAELM.json [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+        "ChangeType": "Addition",
+        "Description": "Added the **accessPackagesIncompatibleWith** navigation property to the [accessPackage](https://docs.microsoft.com/graph/api/resources/accesspackage?view=graph-rest-1.0) resource.",
+        "Target": "accessPackage"
+      },
+      {
+        "Id": "9b5a17bc-343d-4381-a7d3-6cd7224df1f1",
+        "ApiChange": "Relationship",
+        "ChangedApiName": "incompatibleGroups",
+        "ChangeType": "Addition",
+        "Description": "Added the **incompatibleGroups** navigation property on [accessPackage](https://docs.microsoft.com/en-us/graph/api/resources/accessPackage?view=graph-rest-1.0) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IGAELM.json:34]: nit
```suggestion
        "Description": "Added the **incompatibleAccessPackages** relationship to the [accessPackage](https://docs.microsoft.com/graph/api/resources/accesspackage?view=graph-rest-1.0) resource.",
        "Target": "accessPackage"
      },
      {
        "Id": "9b5a17bc-343d-4381-a7d3-6cd7224df1f1",
        "ApiChange": "Relationship",
        "ChangedApiName": "accessPackagesIncompatibleWith",
        "ChangeType": "Addition",
        "Description": "Added the **accessPackagesIncompatibleWith** relationship to the [accessPackage](https://docs.microsoft.com/graph/api/resources/accesspackage?view=graph-rest-1.0) resource.",
        "Target": "accessPackage"
      },
      {
        "Id": "9b5a17bc-343d-4381-a7d3-6cd7224df1f1",
        "ApiChange": "Relationship",
        "ChangedApiName": "incompatibleGroups",
        "ChangeType": "Addition",
        "Description": "Added the **incompatibleGroups** relationship on [accessPackage](https://docs.microsoft.com/en-us/graph/api/resources/accessPackage?view=graph-rest-1.0) resource.",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @markwahl-msft - Thank you for your work on this PR.

Review feedback:
- Surface the additionalAccess method in the methods table of https://learn.microsoft.com/en-us/graph/api/resources/accesspackageassignment?view=graph-rest-1.0
- Document the 7 APIs that relate to the 3 new relationships in acecsspackage.md. Surafce them in the methods table of the accessPackage resource doc and TOC as well.

File: api-reference/v1.0/resources/accesspackage.md [hunk 4]
```diff
@@ -42,6 +49,9 @@ Each access package is referenced by a single access package catalog, and has li
 |:---|:---|:---|
 |assignmentPolicies|[accessPackageAssignmentPolicy](../resources/accesspackageassignmentpolicy.md) collection|Read-only. Nullable.|
 |catalog|[accessPackageCatalog](../resources/accesspackagecatalog.md)|Read-only. Nullable.|
+|incompatibleAccessPackages | [accessPackage](accesspackage.md) collection | The access packages whose assigned users are ineligible to be assigned this access package. |
+|accessPackagesIncompatibleWith | [accessPackage](accesspackage.md) collection | The access packages that are incompatible with this package. Read-only. |
+|incompatibleGroups | [group](group.md) collection | The groups whose members are ineligible to be assigned this access package. |
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/accesspackage.md:54]: nit: alphabetical order for easy scanning

File: api-reference/v1.0/api/accesspackage-delete-incompatibleaccesspackage.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+### Request
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "remove_incompatibleaccesspackage_from_accesspackage"
+}-->
+```http
+DELETE https://graph.microsoft.com/v1.0/identityGovernance/entitlementManagement/accessPackages/{id}/incompatibleAccessPackages/{id}/$ref
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/accesspackage-delete-incompatibleaccesspackage.md:60]: GLOBAL: actual IDs.

**Review by FaithOmbongi — APPROVED** [review]
Approved with suggestions.

File: api-reference/v1.0/resources/accesspackage.md (hunk 4 above)
Comment (markwahl-msft) [reply api-reference/v1.0/resources/accesspackage.md]: updated

**Pull request merged**